scripts/dokka-android.gradle

```diff
@@ -0,0 +1,6 @@
+apply plugin: 'org.jetbrains.dokka-android'
```

Comment:
I think dokka is already included in maven publish plugin (just in case that the build fails).

Reply:
Yes, it is included, but I will leave it here as it will allow to bump independently in the future dokka version.